// @generated/type-graphql/resolvers/outputs/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.TaskSumAggregate = exports.TaskMinAggregate = exports.TaskMaxAggregate = exports.TaskGroupBy = exports.TaskCountAggregate = exports.TaskAvgAggregate = exports.AggregateTask = exports.AffectedRowsOutput = void 0;
var AffectedRowsOutput_1 = require("./AffectedRowsOutput");
Object.defineProperty(exports, "AffectedRowsOutput", { enumerable: true, get: function () { return AffectedRowsOutput_1.AffectedRowsOutput; } });
var AggregateTask_1 = require("./AggregateTask");
Object.defineProperty(exports, "AggregateTask", { enumerable: true, get: function () { return AggregateTask_1.AggregateTask; } });
var TaskAvgAggregate_1 = require("./TaskAvgAggregate");
Object.defineProperty(exports, "TaskAvgAggregate", { enumerable: true, get: function () { return TaskAvgAggregate_1.TaskAvgAggregate; } });
var TaskCountAggregate_1 = require("./TaskCountAggregate");
Object.defineProperty(exports, "TaskCountAggregate", { enumerable: true, get: function () { return TaskCountAggregate_1.TaskCountAggregate; } });
var TaskGroupBy_1 = require("./TaskGroupBy");
Object.defineProperty(exports, "TaskGroupBy", { enumerable: true, get: function () { return TaskGroupBy_1.TaskGroupBy; } });
var TaskMaxAggregate_1 = require("./TaskMaxAggregate");
Object.defineProperty(exports, "TaskMaxAggregate", { enumerable: true, get: function () { return TaskMaxAggregate_1.TaskMaxAggregate; } });
var TaskMinAggregate_1 = require("./TaskMinAggregate");
Object.defineProperty(exports, "TaskMinAggregate", { enumerable: true, get: function () { return TaskMinAggregate_1.TaskMinAggregate; } });
var TaskSumAggregate_1 = require("./TaskSumAggregate");
Object.defineProperty(exports, "TaskSumAggregate", { enumerable: true, get: function () { return TaskSumAggregate_1.TaskSumAggregate; } });
